apps/idos-enclave/src/lib/enclave.js

```diff
@@ -9,6 +9,7 @@ export class Enclave {
   constructor({ parentOrigin }) {
     this.parentOrigin = parentOrigin;
     this.store = new Store();
+    this.authorizedOrigins = JSON.parse(this.store.get("enclave-authorized-origins") ?? "[]");
```

Comment:
Non-actionable: I feel a bit queasy about doing work like this in the constructor. Since we're already grabbing DOM elements a few lines below, I don't think this is making the problem worse.

However, let's keep an eye out for how hard this class is to unit-test.